pkg/transforms/vapbinding.go

```diff
@@ -30,16 +31,24 @@ func VapBindingResourceBuilder(v *unstructured.Unstructured) *VapBindingResource
 
 	owners := v.GetOwnerReferences()
 	fromGK := false
+	fromKyverno := false
 
 	for _, o := range owners {
 		if strings.HasPrefix(o.APIVersion, "constraints.gatekeeper.sh") {
 			fromGK = true
 
 			break
 		}
+
+		if strings.HasPrefix(o.APIVersion, "kyverno.io") {
+			fromKyverno = true
+
+			break
+		}
 	}
 
 	node.Properties["_ownedByGatekeeper"] = fromGK
+	node.Properties["_ownedByKyverno"] = fromKyverno
```

Comment:
Will there ever be other possible owners? A more generic approach could be to combine it into a single property like this
```
node.Properties["_ownedBy"] = "gatekeper"  // or "kyverno"
```

Reply:
That is a good idea Thanks!